docs/user_guide.rst

```diff
@@ -179,7 +179,7 @@ arguments. The following example specifies the log file path:
   import pytest
   @pytest.fixture
   def driver_args():
-      return ['--log-path=chromedriver.log']
+      return ['--log-level=LEVEL']
```

Comment:
You'll need to update preceeding text "The following example specifies the log file path".